web/src/stories/icon.stories.mdx

```diff
@@ -64,6 +92,12 @@ i := component.NewIcon("user", component.WithTooltip("hello", component.TooltipM
 i.Config.Color = "#2596be"
 `}}
 
+export const iconWithSvgDocs = {source: {code: `
+i := component.NewIcon("my-custom-icon")
+svg := <svg width="10px" height="10px"><g><ellipse fill="#000000" stroke="#000000" cx="5" cy="5.5" rx="3" ry="3"/></g></svg>
```

Comment:
What if the width and height are larger sizes? Does the icon scale?

Reply:
It's possible to add oversized icons with svg, the only thing needed is to set the size of the icon.

```
i := component.NewIcon("my-icon")
svg := `<svg width="300" height="200"><polygon points="100,10 40,198 190,78 10,78 160,198"style="fill:lime;stroke:purple;stroke-width:5;fill-rule:evenodd;" /></svg>`
i.SetCustomSvg(svg)
i.SetSize("200")
```

![image](https://user-images.githubusercontent.com/4805305/130679082-9dcaff2a-4947-451a-8c1c-447cff57f89d.png)